src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let prev = initialValue;
+    let startIndex = 0;
+
+    if (arguments.length < 2) {
+      prev = this[0];
+      startIndex = 1;
+    }
```

Comment:
it's a good case to use ternary operator to define the variable

Reply:
fix this 
<img width="580" alt="image" src="https://user-images.githubusercontent.com/99331793/206405047-b7bf97c9-95e2-4bd6-af62-c96a8d3a513d.png">